exchange/exchange.go

```diff
@@ -449,6 +455,19 @@ func (e *exchange) getAllBids(
 			defer func() {
 				e.me.RecordAdapterRequest(bidderRequest.BidderLabels)
 			}()
+
+			if fpdData != nil && fpdData[bidderRequest.BidderName] != nil {
+				if fpdData[bidderRequest.BidderName].Site != nil {
+					bidderRequest.BidRequest.Site = fpdData[bidderRequest.BidderName].Site
+				}
+				if fpdData[bidderRequest.BidderName].App != nil {
+					bidderRequest.BidRequest.App = fpdData[bidderRequest.BidderName].App
+				}
+				if fpdData[bidderRequest.BidderName].User != nil {
+					bidderRequest.BidRequest.User = fpdData[bidderRequest.BidderName].User
+				}
+			}
+
```

Comment:
Can we make this its own function? `getAllBids` is becoming quite large 🙂

Reply:
Yes, done.